### DISCUSSION TITLE
Is it possible to use PlutoSliderServer to host github pages with sliders that work?

### DISCUSSION THREAD

**Discussion by jbrea** [JuliaPluto/PlutoSliderServer.jl#22] [Q&A] [answered]
Currently I publish [the notebooks of my course](https://jbrea.github.io/MLCourse/notebooks/welcome.html) on github pages using the static builder of PlutoSliderServer, following [this recipe](https://github.com/JuliaPluto/static-export-template). That's already cool, but the sliders don't work of course.
I also figured out how to use PlutoSliderServer to host the notebooks with working sliders locally on 127.0.0.1.
But I don't understand how to publish my course in a similar way as the MIT computational thinking course, which has working sliders. Is this possible with just github actions and pages?

**Comment by jbrea** [comment]
Ah, now that I found [this](https://github.com/mitmath/18S191/blob/Spring21/website-docs.md) I guess one needs to run a docker image on e.g. digitalocean.

**Comment by fonsp** [comment, marked as answer]
This is currently not possible, you need to have a running  Julia process to respond to slider changes. We are working on a new idea though...

**Reply by jbrea** [reply]
Oh, cool, I'm looking forward to any new ideas :) In the meantime I found a way to host may course pages on a [university server](https://bio322.epfl.ch).
